packages/core/src/test/codewhisperer/commands/transformByQ.test.ts

```diff
@@ -300,31 +301,14 @@ describe('transformByQ', function () {
 
 describe('resetDebugArtifacts', () => {
     it('should remove the directory containing the pre-build log file if it exists', async () => {
-        const dirPath = await createTestWorkspaceFolder()
-        const preBuildLogFilePath = path.join(dirPath.uri.fsPath, 'DummyLog.log')
-        await toFile('', preBuildLogFilePath)
+        const dirPath = path.join(os.tmpdir(), 'q-transformation-build-logs')
```

Comment:
Have to use `os.tmpdir()` since that's what `resetDebugArtifacts()` uses.

Reply:
why not add an optional `tmpdir` arg to `resetDebugArtifacts` ? Then this becomes less fragile, and more explicit, thus easier to follow.